src/preprocessing/passes/bv_to_int.h

```diff
@@ -48,7 +48,7 @@
  ** Sigma_{i=0}^{b-1}(bvand s[(i+1)*g, i*g] t[(i+1)*g, i*g])*2^(i*g)
  **
  ** More details and examples for this case are described next to
- ** the function createBitwiseNode.
```

Comment:
Actually this line and the one before it can be deleted because this function has moved to another file.

Reply:
Good catch, addressed here c4674e6d2b5717c44ae09a7d317c8c2cc9f2ad3d